app/views/Title.scala

```diff
@@ -18,13 +18,16 @@ package views
 
 import play.api.i18n.Messages
 
-case class Title(title: String)
+case class Title(headingKey: String, sectionKey: Option[String] = None, headingArgs: Seq[String] = Seq.empty) {
```

Comment:
What do you think about use empty string as null object? code would be simple - less pointless `Some` in code

Reply:
I guess so.  Defaulted to "".